generic/piranha/test-resources/kt/feature_flag_system_1/treated/expected/XPFlagCleanerCases.kt

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright (c) 2022 Uber Technologies, Inc.
+
+ <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ except in compliance with the License. You may obtain a copy of the License at
+ <p>http://www.apache.org/licenses/LICENSE-2.0
+
+ <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ express or implied. See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+package com.uber.input
+
```

Comment:
I am not reviewing the input/output behaviors in detail. Assuming they follow the format of Java, and have similar behaviors. We can fix any issues when we apply it internally and detect bad refactorings.